dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -139,7 +141,8 @@ public void setDescription(String description) {
     }
 
     public Set<Entry> getEntries() {
-        return entries.stream().filter(entry -> entry.getIsPublished()).collect(Collectors.toSet());
+        return entries.stream().filter(entry -> entry.getIsPublished()).sorted(Comparator.comparing(Entry::getId)).collect(Collectors.toCollection(
```

Comment:
It's sorting by ID?

Reply:
Is it too weird to sort by name? Is name the 'extra-name' or null if they didn't add one? Or is it the full Dockstore identifier with github and path?

Reply:
We're sorting by entry ID in other places, so keeping it the same.